pootle/apps/pootle_statistics/utils.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from collections import OrderedDict
+
+from django.contrib.auth import get_user_model
+from django.db.models import Count, Q
+from django.utils.functional import cached_property
+
+
+User = get_user_model()
+
+
+class Contributors(object):
+
+    def __init__(self, include_anon=False,
+                 project_codes=None, language_codes=None,
+                 since=None, until=None, sort_by="username"):
```

Comment:
I don't feel it is a good idea to put a mutable type here as default. Better stick to `None` and do some checking before assigning.